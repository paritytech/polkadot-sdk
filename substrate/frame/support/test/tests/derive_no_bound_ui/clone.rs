trait Trait {
	type C;
}

#[derive(frame_support::CloneNoBound)]
struct Foo<T: Trait> {
	c: T::C,
}

fn main() {}
