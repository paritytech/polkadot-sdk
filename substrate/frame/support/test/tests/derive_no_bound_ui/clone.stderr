error[E0277]: the trait bound `<T as Trait>::C: std::clone::Clone` is not satisfied
 --> $DIR/clone.rs:7:2
  |
7 |     c: T::C,
  |     ^ the trait `std::clone::Clone` is not implemented for `<T as Trait>::C`
  |
  = note: required by `std::clone::Clone::clone`
